### DISCUSSION TITLE
文件夹名称显示异常

### DISCUSSION THREAD

**Discussion by opoet** [spencerwooo/onedrive-vercel-index#444] [General]
文件夹名称末尾的大写变成了小写

![image](https://user-images.githubusercontent.com/30021379/154022507-fe90c89e-e3ab-4463-a262-c2b9cc9747b2.png)
![image](https://user-images.githubusercontent.com/30021379/154022718-f16d208b-6568-4d29-9508-42e8d588ba22.png)

当鼠标悬停时显示的文件夹名称正常
![image](https://user-images.githubusercontent.com/30021379/154023444-d5ad84d5-18c5-4112-8364-8cc4aebdf7f1.png)

**Comment by myl7** [comment]
`.` 之后的文件名被判断成扩展名了，而扩展名会被转为小写。我待会儿改一下判断逻辑。

**Comment by myl7** [comment]
See PR #448
